soql-stdlib/src/main/scala/com/socrata/soql/functions/Docs.scala

```diff
@@ -0,0 +1,88 @@
+package com.socrata.soql.functions
+
+import com.socrata.soql.types._
+
+object Docs {
+    def main(args: Array[String]) {
+        val out_path = args(0)
+        SoQLFunctions.allFunctions.filter(x => x.name.toString != "op$[]").foreach { function => 
+            val doc = makeFuncDoc(function)
+            println(function.name.toString)
+            val cleanedName = cleanName(function.name.toString)
+            val file_path = out_path + s"$cleanedName.md"
+            println(file_path)
+            val f = new java.io.File(file_path)
+            if(!f.exists) f.createNewFile
+            val fw = new java.io.FileWriter(f)
+            try fw.write(doc) finally fw.close
+        }
+    }
+    def cleanName(name: String): String = {
+        val rep = name.replace("op$", "").replace("/*", "-*").replace("$", "_")
+        val operators = Map(
+            "+" -> "add",
+            "-" -> "subtract",
+            "*" -> "multiply",
+            "/" -> "divide",
+            "%" -> "modulo",
+            "^" -> "exponent",
+            "||" -> "concatenate",
+            ">" -> "greater_than",
+            "<" -> "less_than",
+            ">=" -> "greater_than_equal",
+            "<=" -> "less_than_equal" ,
+            "==" -> "equal",
+            "=" -> "equal",
+            "!=" -> "not_equal",
+            "<>" -> "not_equal",
+            "or" -> "or",
+            "and" -> "and",
+            "not" -> "not"
+        )
+        operators.getOrElse(rep, rep)
+    }
+    def makeFuncDoc(function: Function[SoQLType]): String = {
+s"""---
+layout: with-sidebar
+sidebar: documentation
+title: "${cleanTitle(function.name.toString)}"
```

Comment:
This is yaml, yes?  `title: ${JString(cleanTitle(...))}` will ensure everything gets escaped properly.

Reply:
(same pretty much everywhere you're splicing in stuff, converting to a `JValue` of some kind will produce valid yaml and ensure everything is properly quoted and escaped)